src/megatron/energon/__init__.py

```diff
@@ -1,6 +1,14 @@
 # Copyright (c) 2025, NVIDIA CORPORATION.
 # SPDX-License-Identifier: BSD-3-Clause
 
+try:
+    from importlib.metadata import version
+
+    __version__ = version("megatron-energon")
+except Exception:
+    # Fallback for development mode when package is not installed
+    __version__ = "unknown"
```

Comment:
shouldn't we fallback to the git version? I.e. try to fetch the last tag from git, and add the number of commits on top.

Reply:
As a fallback of the fallback, maybe we should default to `1.0.0` for datasets which don't have a version?